instrumentation/logger.go

```diff
@@ -13,6 +13,24 @@ import (
 	"os"
 )
 
+func GetBootstrapCrashLogger() log.Logger {
+	path := "./orbs-network-bootstrap.log"
+
+	logFile, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		panic(err)
+	}
+
+	fileWriter := log.NewTruncatingFileWriter(logFile)
+	outputs := []log.Output{
+		log.NewFormattingOutput(fileWriter, log.NewHumanReadableFormatter()),
+		log.NewFormattingOutput(os.Stdout, log.NewHumanReadableFormatter()),
+		log.NewFormattingOutput(os.Stderr, log.NewHumanReadableFormatter()),
```

Comment:
do we need both? aren't both being mapped to docker logs?

Reply:
I'm not sure exactly what output goes where in prod. and I suspect it's a good thing not being able to assume that in the bootstrap phase (only prior to logger init): there should be 0 lines, so write them to all directions. better have 3 copies of a never-should-happen critical error, than none.